org.kde.okular.json

```diff
@@ -287,6 +287,23 @@
             }
          ]
       },
+      {
+         "name": "okular-wrapper",
+         "buildsystem": "simple",
+         "build-commands": [
+            "install -Dm755 okular-wrapper /app/bin/okular-wrapper"
+            ],
+         "sources": [
+            {
+                "type": "script",
+                    "dest-filename": "okular-wrapper",
+                    "commands": [
+                        "export TMPDIR=\"$XDG_RUNTIME_DIR/app/$FLATPAK_ID\"",
+                        "exec /app/bin/okular \"$@\""
```

Comment:
it can be in one line with `exec env TMPDIR=$XDG_CACHE_HOME /app/bin/okular \"$@\"`

Reply:
I know but I don't like that. I think my approach is cleaner.